src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -301,32 +283,24 @@ private boolean validateTargetPort(String targetPort) {
      * screen.
      */
     public FormValidation doCheckTargetPort(@QueryParameter("targetPort") final String targetPort) {
-        if (!validateTargetPort(targetPort)) {
+        if (!validatePort(targetPort)) {
```

Comment:
To be consistent here, we could inline `validatePort` in that if block.

EDIT: nervermind, that code is reused